### PR TITLE
using new logo and updating layout to accommodate it

### PR DESCRIPTION
**Pull request by blairconrad**
The "use the new logo" part of #5.

I can do the favicon part in another PR or another commit.

I was easily distracted while updating the CSS, so there are a few "bonuses" mixed in, mostly around the padding and layout.
If you want, I can try to tease them back apart and resubmit a multi-commit PR. But not today.

### CODE REVIEW

**Comment by adamralph** [comment]
re https://github.com/FakeItEasy/FakeItEasy/issues/391

**Comment by adamralph** [comment]
Do you know any easy way to test the changes? I.e. get the site to render properly and eyeball it?

File: index.html [hunk 1]
```diff
@@ -11,21 +11,17 @@
...
   <body>
     <div class="splash">
-      <div class="box splash-text">
-        <img class="floatright" src="img/talkingladies.png" alt="Two ladies talking about FakeItEasy">
-        <h1 class="splash-head">FakeItEasy</h1>
-        <h2 class="splash-subhead">The easy mocking library for .NET</h2>
-
-      </div>
+      <img src="img/fakeiteasy_logo_256.png" alt="FakeItEasy logo">
+      <h2>The easy mocking library for .NET</h2>
```
Comment (adamralph) [inline index.html]: We should probably be using a 512 image, sized to 256, to support high DPI scaled up to 200%. Or perhaps, we should think about switching to SVG. (Of course, you can't actually do any of this until I've uploaded the source images to our new repo.)

File: index.html [hunk 2]
```diff
@@ -11,21 +11,17 @@
...
   </head>
   <body>
     <div class="splash">
-      <div class="box splash-text">
-        <img class="floatright" src="img/talkingladies.png" alt="Two ladies talking about FakeItEasy">
-        <h1 class="splash-head">FakeItEasy</h1>
-        <h2 class="splash-subhead">The easy mocking library for .NET</h2>
-
-      </div>
+      <img src="img/fakeiteasy_logo_256.png" alt="FakeItEasy logo">
```
Comment (adamralph) [inline index.html]: We should probably be using a 512 image, sized to 256, to support high DPI scaled up to 200%. Or perhaps, we should think about switching to SVG. (Of course, you can't actually do any of this until I've uploaded the source images to our new repo.)

Comment (blairconrad) [inline index.html]: I like the SVG idea, personally. I've not kept up with browser support. Otherwise, the larger image scaled down would work too.

**Comment by blairconrad** [comment]
> Do you know any easy way to test the changes? I.e. get the site to render properly and eyeball it?

I'm not sure I follow. I was having jekyll serve locally for me.
I used Chrome developer tools to easily help me simulate smaller screens, but really just resizing the browser did a pretty good job of that.

If you want an easier way to view it, I don't mind throwing the site up as a subdir of my personal site.

**Comment by adamralph** [comment]
I tried using jekyll to server it locally but it blows up horribly. I think it's to do with gem versions. I tried adding a Gemfile and using bundler as recommended at https://help.github.com/articles/using-jekyll-with-pages/ but that doesn't work either. I think something to do with https://github.com/jekyll/jekyll/issues/2789. Yet another episode in the 'ruby on windows' saga. I'll keep playing around to see if I can get it working. If you could host it somehow that would be great, yes.

**Comment by adamralph** [comment]
OK, I've finally got it working, after installing Python, of all things.

**Comment by blairconrad** [comment]
For pygments?

**Comment by blairconrad** [comment]
(Sorry, I stepped away to participate in the roasting of Brussels Sprouts.)

**Comment by adamralph** [comment]
Yeah, pygments needed it.

If we want to hold off the website changes until we go live with 2.0, you'll have to add the favicon changes to this PR. If we're happy for the website to go "2.0" beforehand, then I can merge this PR now and the favicon one later. I really don't mind which approach we take.

**Comment by blairconrad** [comment]
Well, the README has it now. I say let's go for it.

**Pull request merged**